### DISCUSSION TITLE
【未解决】ArrayTable属性无效bug

### DISCUSSION THREAD

**Discussion by caixianglin** [alibaba/formily#2750] [Q&A 2.x]
链接：https://codesandbox.io/s/hungry-hugle-8pxeg?file=/App.tsx
自增表格列配置，锁列，尺寸等属性都无效

锁列无效
![image](https://user-images.githubusercontent.com/10279815/148895485-4306c657-6d96-4edc-97b6-ba658afba08a.png)

**Comment by janryWang** [comment]
json能不能简化一下，太复杂了，看不出具体问题呢

**Reply by caixianglin** [reply]
已简化，select好像可以，但是锁列这些属性还是无效

**Comment by janryWang** [comment]
把ArrayTable的style.width 100%干掉就行了

**Reply by caixianglin** [reply]
![image](https://user-images.githubusercontent.com/10279815/149075920-fa9fee6a-d284-4a64-b6fc-c94b6e1c8942.png)
无效啊，哪里操作不对吗
